logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc

```diff
@@ -0,0 +1,536 @@
+// WARNING - DO NOT ALTER THE URL PATH OF THIS CONTENT, OR YOU WILL BREAK LINKS FROM ALERT MESSAGES THAT LINK TO THIS CONTENT.
```

Comment:
Thanks - Because the software links directly to this content, it is _critical_ that writers don't alter the URL without coordinating such a change with the engineering team. I am using all caps because it increases the likelihood that writers  will notice this information.